vk/vk/Model/VKServiceAPIManager.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  VKServiceAPIManager.swift
+//  vk
+//
```

Comment:
```suggestion
//
 //  Created by Pavel on 22.11.22.
 //
```

форматер должен был сработать